html/php-pre-challenge1/index.php

```diff
@@ -1,5 +1,17 @@
 <?php
 for ($i = 1; $i <= 100; $i++) {
-    //ここからコードを書く
-    echo $i;
+    if($i % 3 == 0 && $i % 5 == 0){
```

Comment:
```suggestion
    if($i % 15 === 0){
```
こう書いた方が低負荷です。
JSは各々のクライアントのCPUリソースを使うのでそこまで問題になりませんが、サーバサイドスクリプトは極力サーバへの負荷を抑えるべきとされています。